### DISCUSSION TITLE
Issues from 3/6

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#240] [General]
It occurred to me what was wrong with our cone placement auton.

We don't have a completion criteria on our "delay" input, so it is looking at the other criteria and deciding the step is complete. We need to add some completion criteria for "delaying". We also need to evaluate the encoder "deadband" as I think it is small enough that when we try to move from "approach" to "high" the code says it is done too soon. If we fix those two things, I think that at least the motion will be corrected.

**Comment by programmeruser517** [comment]
This makes sense, I'll see if I can edit this before we start today. Thanks!
